selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromCentosNodeStackTest.java

```diff
@@ -81,7 +82,7 @@ public void checkWorkspaceCreationFromCentosNodeStack() {
     projectExplorer.waitProjectInitialization(WEB_NODE_JS_PROJECT);
   }
 
-  @Test(priority = 1)
+  @Test(priority = 1, groups = UNDER_REPAIR)
```

Comment:
It's expected that test under repair points to issue which describe test failure, in try-catch block around failing test command.